src/hip/hiperrors.cpp

```diff
@@ -30,7 +30,7 @@ namespace miopen {
 
 std::string HIPErrorMessage(int error, const std::string& msg)
 {
-    return msg + " " + hipGetErrorString(static_cast<hipError_t>(error));
+    return msg + ": " + hipGetErrorString(static_cast<hipError_t>(error));
```

Comment:
is ":" change mandatory?

Reply:
Yes, if we want to separate error message from the error code, see https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1729/files#r962057473

Reply:
ok.